### ISSUE TITLE
OurWebElement and WebElement

### ISSUE THREAD

**Issue by skopanirov** [WileyLabs/teasy#13]
Hi all!

Teasy uses OurWebElement class instead of classic Selenium WebElement class, but there are some functions in OurWebElement.class that still return classic Selenium WebElement:

```
public WebElement findElement(By by)
public List<WebElement> findElements(By by)
```

This approach causes some issues with incompatible types error. Will it be fixed?

**Comment by vefimofff** [comment]
I quickly checked and it seems that most likely we will delete this methods in future. Currently they're used in the deprecated code so for now I will do the following.
1. mark these methods as deprecated
2. will see if it's possible to make them package-private, seems like it is.

**Comment by skopanirov** [comment]
Well, will it be available any other way to find an element inside another element?

**Comment by vefimofff** [comment]
Yes, it is already possible. Check methods like element(), domElement(), and elementOrNull() of OurWebElement.

**Comment by skopanirov** [comment]
Ok, thank you!

**Comment by vefimofff** [comment]
fixed in 1.0.2-beta
